plugins/sql/README.md

```diff
@@ -158,7 +158,26 @@ fn main() {
 
 ### Applying Migrations
 
-Migrations are applied automatically when the plugin is initialized. The plugin runs these migrations against the database specified by the connection string. Ensure that the migrations are defined in the correct order and are idempotent (safe to run multiple times).
+To apply the migrations when the plugin is initialized, add the connection string to the `tauri.conf.json` file:
+
+```json
+{
+  "plugins": {
+    "sql": {
+      "preload": ["sqlite:mydatabase.db"]
+    }
+  }
+}
+```
+
+Alternatively, the client side `load()` also runs the migrations for a given connection string:
+
+```ts
+import Database from "@tauri-apps/plugin-sql";
+const db = await Database.load("sqlite:mydatabase.db");
+```
+
+Ensure that the migrations are defined in the correct order and are idempotent (safe to run multiple times).
```

Comment:
Can we just replace the word with the explanation or something? 😅
Just
```
Ensure that the migrations are defined in the correct order and are safe to run multiple times.
```
sounds totally fine to me

Reply:
Yep, makes sense. Fixed in e8d0630.